wolfcrypt/src/sha256.c

```diff
@@ -1752,7 +1752,7 @@ void wc_Sha256Free(wc_Sha256* sha256)
          * should have already been released (lockDepth = 0)
          */
         InitSha256(sha256); /* unlock mutex, set mode to ESP32_SHA_INIT */
-        ESP_LOGE("sha256", "ERROR: hardware unlock needed in wc_Sha256Free");
+        ESP_LOGV("sha256", "ERROR: hardware unlock needed in wc_Sha256Free");
```

Comment:
This indicates "ERROR" but is flagged as verbose. Please fix the message or leave as LOGE.

Reply:
The text of the message has been updated.